interface/main/onotes/office_comments.php

```diff
@@ -32,18 +25,20 @@
 <head>
 
 <?php Header::setupHeader(); ?>
-
+<title><?php echo xlt('Office Notes'); ?></title>
 </head>
 <body class="body_top">
 
-<div id="officenotes_list">
-<a href="office_comments_full.php" onclick='top.restoreSession()'>
-<font class="title"><?php echo xlt('Office Notes'); ?></font>
-<font class="more"><?php echo text($tmore);?></font></a>
-
-<br>
-
-<table border=0 width=100%>
+<div class="container">
+   
+   <div class="row">
+      <div class="col-sm-6">
+         <div class="form-group form-horizontal page-header">
+            <h3><label class='col-sm-4 text-right'><?php echo xlt('Office Notes'); ?></label></h3>
+            <p class="more control-label col-sm-1"><a href="office_comments_full.php" onclick='top.restoreSession()'><?php echo text($tmore); ?></a></p>
```

Comment:
btn-link is not in themes/core.css and I do not find a btn-more or something like that. I will us btn-show.